src/components/Footer/footer.tsx

```diff
@@ -1,13 +1,59 @@
 import * as React from "react";
-import "./footer.css";
+// import "./footer.css";
+
+import styled from "styled-components";
+
+const FooterContainer = styled.footer`
+  height: 100px;
+  width: 100vw;
```

Comment:
As a default, when calculating viewport width, the browser disregards the scrollbar, so setting something as 100vw can actually be like setting something as 100% + 12-20px which might lead to scenarios where you can scroll horizontally when you didn't intend to